### PR TITLE
Ignore symlink flag

### PR DESCRIPTION
**Pull request by rpmoore**

### CODE REVIEW

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java [hunk 1]
```diff
@@ -302,6 +305,8 @@ public void getBulkObjectWithSync() throws Exception {
 
     @Test
     public void putObjectWithSync() throws Exception {
+        assumeThat(Util.getBlackPearlVersion(client), greaterThan(3.0));
```
Comment (shabtaisharon) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: this test can be run on bp >= 3.0. Does greaterThan check for equal?

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java [hunk 2]
```diff
@@ -327,6 +326,8 @@ public void putObjectWithSync() throws Exception {
 
     @Test
     public void putBulkObjectWithSync() throws Exception {
+        assumeThat(Util.getBlackPearlVersion(client), greaterThan(3.0));
+
```
Comment (shabtaisharon) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: same question

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java [hunk 3]
```diff
@@ -375,6 +371,8 @@ public void putBulkObjectWithPipe() throws Exception {
 
     @Test
     public void putBulkObjectWithPipeAndSync() throws Exception {
+        assumeThat(Util.getBlackPearlVersion(client), greaterThan(3.0));
+
```
Comment (shabtaisharon) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: same question

File: ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java (hunk 1 above)
Comment (rpmoore) [inline ds3-cli-integration/src/test/java/com/spectralogic/ds3cli/integration/FeatureIntegration_Test.java]: Ohhh, good point.  Let me fix this.

**Pull request merged**